assignment/tool/src/webapp/vm/assignment/chef_assignments_instructor_student_list_submissions.vm

```diff
@@ -173,7 +173,9 @@
 													#set ($submissionSubmitter = $!service.getSubmissionSubmittee($submission))
 													#if ($!submissionSubmitter.isPresent())
 														#set ($submitterId = $!submissionSubmitter.get().getSubmitter())
-														<br />$tlang.getString("listsub.submitted.by") $studentMembersMap.get($submitterId).getDisplayName()
+														#if ($!submissionSubmitter.get().getSubmittee())
```

Comment:
If the submitter is not in the map this is probably some other issue going on... Maybe adding some an else indicating that would be ideal?

Reply:
@jesusmmp can we add an "else"?

Reply:
I left comments on the JIRA but I could not replicate this on master, can you please provide the steps.